addons/editor-stepping/userscript.js

```diff
@@ -15,7 +15,7 @@ export default async function ({ addon, global, console }) {
       </feFuncA>
     </feComponentTransfer>
 
-    <feFlood flood-color="blue" flood-opacity="1" result="outColor">
+    <feFlood flood-color="` + addon.settings.get("highlight-color")  + `" flood-opacity="1" result="outColor">
```

Comment:
Do we need to worry about xss here?

Reply:
Yes.

Reply:
It's just a color input, however modifying the config file could do something here. But I don't think it's such a big thing to worry about

Reply:
We won't merge this until this is fixed.